.github/workflows/workflow.yml

```
@@ -0,0 +1,64 @@
+  pull_request:
+    types: [opened]
+    branches-ignore:
+      - main
+    pull_request_review_comment:
+      types: [created]
+    issue_comment:
+      types: [created]
+
+concurrency:
+  group:
+    ${{ github.repository }}-${{ github.event.number || github.head_ref || github.sha }}-${{ github.workflow }}-${{ github.event_name = 'pull_request_review_comment' && 'pr_comment' || 'pr' }}
```

actionlint (1.7.7)

got unexpected character ' ' while lexing == operator, expecting '='

(expression)